docs/ibc/upgrades/developer-guide.md

```diff
@@ -1,9 +0,0 @@
-<!--
```

Comment:
can't seem to find where this was moved. It might have been deleted alltogether since it was just pointing to another location?